Document-Processing/Excel/Spreadsheet/Javascript-ES6/notes.md

```diff
@@ -32,38 +32,100 @@ In the below example, you can add, edit, save, and delete notes.
 
 In the active worksheet, you can add a note in the following ways:
 
-* To add a note, right-click the cell to open the context menu and choose the **"Add Note"** option from the context menu. This will open a dialog box to add the content as a note.
-* You can also use the `Shift` + `F2` keyboard shortcut to add a note to the desired cell. A dialog box will be opened to add the content as a note.
-* After entering the content in the dialog box, you can either click on other cells or press the `Esc` button on the keyboard to automatically save the note in the cell and close the dialog box.
+* **Context Menu**: Right-click on the desired cell and select **Add Note**.
+* **Ribbon**: Select the cell, navigate to the **Review** tab, click the **Notes** dropdown, and select **Add Note**.
+* **Keyboard Shortcut**: Select the cell and press `Shift` + `F2`.
+
+A dialog box will open where you can enter the note content. After entering the content, you can either click on other cells or press the `Esc` button to automatically save the note and close the dialog box.
```

Comment:
A note editor popup opens where you can enter the content. To save and close the popup, click anywhere outside the editor or press `Esc`; the note is saved automatically.

Use keyboard tag instead of bold or   for "Esc".
Refer - https://help.syncfusion.com/document-processing/excel/spreadsheet/blazor/accessibility#keyboard-shortcuts